_posts/2021-05-28-seqan-3-0-3-release.md

```diff
@@ -0,0 +1,130 @@
+* Check out our updated [SeqAn3 Cookbook](https://docs.seqan.de/seqan/3.0.3/cookbook.html). It contains a listing of code
+  examples on how to perform particular tasks using the library.
+
+Note that 3.1.0 will be the first API stable release and interfaces in this release might still change.
+
+### Our final push to 3.1.0
+
+For 3.0.3, we heavily focused on making the final push towards 3.1.0 and took our time to really define the scope of our Sequence Analysis Library.
+
+That means the next Release (3.1.0) will be the first stable one, exciting right? :partying_face:
+
+### :open_book: Module structure
+
+We now utilise the following module structure:
+
+#### Sequence Analysis Modules
+
+* `#include <seqan3/alphabet/*.hpp>`:
+  Contains alphabet related entities, like `seqan3::dna4`.
+* `#include <seqan3/alignment/*.hpp>`:
+  Contains sequence alignment related entities, like `seqan3::align_pairwise`.
+* `#include <seqan3/io/*.hpp>`:
+  Contains I/O related entities, like `seqan3::sequence_file_input`.
+* `#include <seqan3/search/*.hpp>`:
+  Contains search related entities, like `seqan3::search`.
+
+We also moved containers and views from the former range module into related Sequence Analysis Module (e.g., `seqan3::bitpacked_sequence` and `seqan3::translate` are now part of the alphabet module).
+
+All Sequence Analysis Modules will have strong [API Stability](https://docs.seqan.de/seqan/3.0.3/about_api.html#api_stability) guarantees. So far we finished declaring the stability in our documentation for alphabets, containers, and ranges. Sequence Alignment and I/O will follow in the next release.
+
+#### General Purpose Modules
+
+* `#include <seqan3/argument_parser/*.hpp>`:
+  Our argument parser library that helps to write apps.
+* `#include <seqan3/range/*.hpp>` (split since 3.0.3):
+  We moved most entities within this module into the other modules: alphabet, alignment, io, search or utility. But, we also deprecated some of them that did not fit our scope anymore.
+* `#include <seqan3/core/*.hpp>`:
+  Our internal module for entities that are shared across our sequence analysis modules and are needed to implement our own algorithms and data structures.
+* `#include <seqan3/utility/*.hpp>`:
+  utility contains entities that are completely unrelated to biology-problems. We think of these as something that could be made separate libraries.
+
+All General Purpose Modules will have no [API Stability](https://docs.seqan.de/seqan/3.0.3/about_api.html#api_stability) guarantees.
+
+### :lock: API Stability
+
+We are especially thrilled to announce 3.0.3, because this release should be the first one that _just compiles_<sup>:tm:</sup> your app when upgrading from 3.0.2 to 3.0.3. Arguably, you will encounter scattered "deprecation" notices, but all of those messages should point you to an upgrade path. Unless you treat warnings as errors, your app will still compile even when encountering deprecation notices.
```

Comment:
```suggestion
We are especially thrilled to announce 3.0.3, because this release should be the first one that _just compiles_<sup>™️</sup> your app when upgrading from 3.0.2 to 3.0.3. Arguably, you will encounter scattered "deprecation" notices, but all of those messages should point you to an upgrade path. Unless you treat warnings as errors, your app will still compile even when encountering deprecation notices.
```